tests/system/python/conftest.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Configuration system/python/conftest.py
+
+"""
+import pytest
+import subprocess
+import os
+
+__author__ = "Vaibhav Singhal"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+@pytest.fixture
+def reset_foglamp_start():
+    assert os.environ.get('FOGLAMP_ROOT') is not None
```

Comment:
should we check what should be the exact value

Reply:
We cant, as we are not sure where will the end user clone foglamp, it can be /home/ubuntu/FogLAMP or /home/bla/Development/FogLAMP